bin/edit.js

```diff
@@ -15,17 +15,31 @@ module.exports = Cli.createCommand('edit', {
             required: false
         },
     },
+    optionGroups: {
+        'Output options': {
+            'version': {
+                alias: 'v',
+                description: 'Use the previous version of the editor',
+                dest: 'version',
+                choices: ['v1', 'v2'],
+                type: 'string',
+            },
+        }
+    },
     handler: handleEdit,
 });
 
 
 // Command handler
-
 function handleEdit(args) {
     var profile = args.profile;
     var wtName  = args.name ? args.name + '/' : '';
     var url     = profile.url + '/edit/' + profile.container + '#/' + wtName + profile.token;
 
+    if (args.version) {
+        url = profile.url + (args.version === `v1` ? '/prev' : '') + '/edit/' + profile.container + '#/' + wtName + profile.token;
```

Comment:
I agree w/ @tjanczuk that `/prev` will not scale and that absolute versions would be a better solution.